plugins/woocommerce/src/Enums/ProductVisibility.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Enums;
+
+/**
+ * Enum class for all the product visibility.
+ */
+class ProductVisibility {
```

Comment:
Elsewhere in the codebase, but also I think in the docs, we tend to speak of *catalog* visibility. To that end, what do you think of renaming this class as `CatalogVisibility`? Feels like that might align a little more tightly.

Reply:
+1, I think matching the name of the class up with the name of the product property will make it easier to reason about the code.

Reply:
Oh right, renamed it in https://github.com/woocommerce/woocommerce/pull/54001/commits/5c1204ea361cfb5aeb9957c472900706eabda3c5